ui/css/design-system/typography.scss

```diff
@@ -48,27 +50,34 @@ $fa-font-path: 'fonts/fontawesome';
 }
 
 @font-face {
-  font-family: 'Euclid';
+  font-family: 'Euclid Circular B';
   font-style: normal;
   font-weight: 400;
   src: url('fonts/Euclid/EuclidCircularB-Regular-WebXL.ttf') format('truetype');
 }
 
 @font-face {
-  font-family: 'Euclid';
+  font-family: 'Euclid Circular B';
   font-style: italic;
   font-weight: 400;
   src: url('fonts/Euclid/EuclidCircularB-RegularItalic-WebXL.ttf') format('truetype');
 }
 
 @font-face {
-  font-family: 'Euclid';
+  font-family: 'Euclid Circular B';
+  font-style: normal;
+  font-weight: 500;
+  src: url('fonts/Euclid/EuclidCircularB-Medium.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Euclid Circular B';
```

Comment:
Can we at least use some of the design tokens for typography here?

Reply:
Yeah maybe we should have added the `sans-serif` as another token and used it like 
```
  --font-family-euclid-circular-b: 'Euclid Circular B';
  --font-family-roboto: 'Roboto';
  --font-family-sans-serif: sans-serif;

  --typography-s-display-md-font-family: var(--font-family-euclid-circular-b) var(--font-family-sans-serif);
```

Reply:
Ah that wouldn't work either https://stackoverflow.com/questions/53846582/can-i-use-css-variables-in-a-font-list-and-have-it-work-in-legacy-browsers